gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -301,13 +301,13 @@ TFLiteCore::invoke (uint8_t * inptr, uint8_t ** outptr)
 
   int sizeOfArray = NNS_TENSOR_RANK_LIMIT;
 
-  for (int i = 0; i < sizeOfArray; i++) {
-    output_number_of_pixels *= inputTensorMeta.dims[0][i];
-  }
-
   for (int i = 0; i < getInputTensorSize (); i++) {
     int input = interpreter->inputs ()[i];
 
+    for (int j = 0; j < sizeOfArray; j++) {
+      output_number_of_pixels *= inputTensorMeta.dims[i][j];
```

Comment:
Looks like incorrect.

If there are two tensors in one ```other/tensors``` with 5:5, the expected number of pixels is 50, not 625.

Reply:
You're right. I'll fix it.